examples/complete/kubectl.tf

```diff
@@ -16,26 +16,26 @@ locals {
 }
 
 resource "local_file" "kubeconfig" {
-  count    = "${var.enabled == "true" && var.apply_config_map_aws_auth == "true" ? 1 : 0}"
-  content  = "${module.eks_cluster.kubeconfig}"
-  filename = "${local.kubeconfig_filename}"
+  count    = var.enabled == "true" && var.apply_config_map_aws_auth == "true" ? 1 : 0
+  content  = module.eks_cluster.kubeconfig
+  filename = local.kubeconfig_filename
 }
 
 resource "local_file" "config_map_aws_auth" {
-  count    = "${var.enabled == "true" && var.apply_config_map_aws_auth == "true" ? 1 : 0}"
-  content  = "${module.eks_workers.config_map_aws_auth}"
-  filename = "${local.config_map_aws_auth_filename}"
+  count    = var.enabled == "true" && var.apply_config_map_aws_auth == "true" ? 1 : 0
+  content  = module.eks_workers.config_map_aws_auth
+  filename = local.config_map_aws_auth_filename
 }
 
 resource "null_resource" "apply_config_map_aws_auth" {
-  count = "${var.enabled == "true" && var.apply_config_map_aws_auth == "true" ? 1 : 0}"
+  count = var.enabled == "true" && var.apply_config_map_aws_auth == "true" ? 1 : 0
 
   provisioner "local-exec" {
-    command = "kubectl apply -f ${local.config_map_aws_auth_filename} --kubeconfig ${local.kubeconfig_filename}"
+    command = "${local.set_context} && kubectl apply -f ${local.config_map_aws_auth_filename}"
```

Comment:
This avoided an 'authorized' message and does the exact same functionality. Potentially can remove 'config_map_aws_auth_filename' since it isn't being used.